src/content/app/genome-browser/components/browser-track-config/services/browser-track-config-storage-service.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import storageService, {
+  StorageServiceInterface
+} from 'src/services/storage-service';
+import {
+  GenomeTrackConfigs,
+  TrackConfigsPerGenome
+} from 'src/content/app/genome-browser/state/track-config/trackConfigSlice';
+
+export enum StorageKeys {
+  TRACK_CONFIG_STATES = 'browser.trackConfigStates'
```

Comment:
Throughout this file — the word "state" is redundant

Reply:
✅